### PR TITLE
drivers: i3c: rename dcr i2c mode macro to lvr

### PR DESCRIPTION
**Pull request by XenuIsWatching**
This renames the I2C 'DCR' mode to 'LVR' as that is the variable it
should be looking at and not the dcr value. This also fixes the get
'lvr' mode argument.

### CODE REVIEW

**Review by dcpleung — DISMISSED** [review]

**Review by dkalowsk — CHANGES_REQUESTED** [review]
Nothing wrong with the change as is, just an additional request.  This is going to break an API for folks on earlier revisions, so... requesting that you add a note in the release notes now as this comes in.

**Comment by XenuIsWatching** [comment]
> Nothing wrong with the change as is, just an additional request. This is going to break an API for folks on earlier revisions, so... requesting that you add a note in the release notes now as this comes in.

done

**Review by dcpleung — DISMISSED** [review]

**Review by dkalowsk — DISMISSED** [review]
Thanks!

File: doc/releases/release-notes-3.6.rst [hunk 1]
```diff
@@ -145,6 +145,9 @@ Drivers and Sensors
 
 * I3C
 
+  * The Legacy Virtual Register defines have been renamed from I3C_DCR_I2C_* to
+    I3C_LVR_I2C_*.
```
Comment (henrikbrixandersen) [inline doc/releases/release-notes-3.6.rst:149]: You need to enclose the names in double backticks.

**Review by henrikbrixandersen — DISMISSED** [review]

File: doc/releases/release-notes-3.6.rst (hunk 1 above)
Comment (XenuIsWatching) [reply doc/releases/release-notes-3.6.rst]: done

**Review by dcpleung — APPROVED** [review]

**Review by dkalowsk — APPROVED** [review]

**Pull request merged**